src/Kygekraqmak/KygekPingTPS/TPSCommand.php

```diff
@@ -21,40 +21,43 @@
  * (at your option) any later version.
  *
 */
-
 declare(strict_types=1);
 
 namespace Kygekraqmak\KygekPingTPS;
 
+use pocketmine\Player;
 use pocketmine\utils\Config;
+use pocketmine\utils\TextFormat as TF;
 use pocketmine\command\CommandSender;
 use pocketmine\command\Command;
-use pocketmine\utils\TextFormat as TF;
-
-class TPS {
-
-	public $tps;
 
-    private function getConfig() : Config {
-        return Main::getInstance()->getConfig();
-    }
+class TPSCommand extends Command {
+	
+	/** @var Main $plugin */
+	protected $plugin;
 
-	public function TPSCommand(CommandSender $sender, Command $cmd, string $label, array $args) {
-		$this->tps = Main::getInstance()->getServer()->getTicksPerSecond();
-		if ($sender->hasPermission("kygekpingtps.tps")) $sender->sendMessage($this->getServerTPSMessage());
-		else $sender->sendMessage($this->getNoPermMessage());
+	public function __construct(Main $plugin) {
+		$this->plugin = $plugin;
+		parent::__construct("tps", "Current tps of a player", "");
+		$this->setPermission("kygektpstps.tps");
 	}
 
-    private function getNoPermMessage() : string {
-        $noperm = $this->getConfig()->get("no-permission", "");
-        $noperm = Main::replace($noperm);
-        return empty($noperm) ? Main::PREFIX . TF::RED . "You do not have permission to use this command" : $noperm;
-    }
-
-    private function getServerTPSMessage() : string {
-        $servertps = $this->getConfig()->get("server-tps", "");
-        $servertps = str_replace("{tps}", (string) $this->tps, Main::replace($servertps));
-        return empty($servertps) ? Main::PREFIX . TF::GREEN . "Current server TPS: " . TF::AQUA . $this->tps : $servertps;
+	private function getConfig() : Config {
+	    return $this->plugin->getConfig();
     }
-
-}
+	
+	public function execute(CommandSender $sender, string $commandLabel, array $args){
+		if(!$this->testPermission($sender)){
+			$sender->sendMessage(Main::replace($this->getConfig()->get("no-permission", "")));
+			return false;

```

Comment:
You can use `return true` instead, unless there's a good reason to use `return false`.